Python/pywarpx/_libwarpx.py

```diff
@@ -15,8 +15,7 @@
 
 import numpy as np
 from numpy.ctypeslib import ndpointer as _ndpointer
-
-from .Geometry import geometry
+import pywarpx
```

Comment:
My preference is that a package should not import itself, so I would keep `from .Geometry import geometry` and add `from . import fields`.

Reply:
I just saw your comment. I don't think the circular import is a problem. BTW, having _libwarpx.py import pywarpx is also circular since pywarpx imports _libwarpx.py.

Reply:
Unfortunately that doesn't work as a global import (see https://stackoverflow.com/questions/7336802/how-to-avoid-circular-imports-in-python). However, I could do the import inside `depositChargeDensity`, would you prefer that?

Reply:
Just for reference, doing the `from . import fields` I get the error trace:
```
Traceback (most recent call last):
  File "PICMI_inputs_1d.py", line 11, in <module>
    from pywarpx import callbacks, fields, picmi
  File "/home/roelof/repository/ME_WarpX/Python/pywarpx/__init__.py", line 19, in <module>
    from .WarpX import warpx
  File "/home/roelof/repository/ME_WarpX/Python/pywarpx/WarpX.py", line 22, in <module>
    from ._libwarpx import libwarpx
  File "/home/roelof/repository/ME_WarpX/Python/pywarpx/_libwarpx.py", line 20, in <module>
    from . import fields
  File "/home/roelof/repository/ME_WarpX/Python/pywarpx/fields.py", line 25, in <module>
    from ._libwarpx import libwarpx
ImportError: cannot import name 'libwarpx' from 'pywarpx._libwarpx' (/home/roelof/repository/ME_WarpX/Python/pywarpx/_libwarpx.py)
```

Reply:
Ah yes, the `libwarpx` object needs to be created first before in can be imported into fields. So yes, I think the solution is to import fields in `depositChargeDensity`.

Reply:
Sounds good!